protos/local/workflow_commands.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
+
+package coresdk.workflow_commands;
+
+import "common.proto";
+
+message WorkflowCommand {
+    oneof variant {
+        StartTimer start_timer = 1;
+        ScheduleActivity schedule_activity = 2;
+        QueryResult respond_to_query = 3;
+        RequestActivityCancellation request_activity_cancellation = 4;
+        CompleteWorkflowExecution complete_workflow_execution = 5;
+        FailWorkflowExecution fail_workflow_execution = 6;
+
+        // To be added as/if needed:
+        //  RequestCancelActivityTask request_cancel_activity_task_command_attributes = 6;
+        //  CancelWorkflowExecution cancel_workflow_execution_command_attributes = 8;
+        //  RequestCancelExternalWorkflowExecution request_cancel_external_workflow_execution_command_attributes = 9;
+        //  RecordMarker record_marker_command_attributes = 10;
+        //  ContinueAsNewWorkflowExecution continue_as_new_workflow_execution_command_attributes = 11;
+        //  StartChildWorkflowExecution start_child_workflow_execution_command_attributes = 12;
+        //  SignalExternalWorkflowExecution signal_external_workflow_execution_command_attributes = 13;
+        //  UpsertWorkflowSearchAttributes upsert_workflow_search_attributes_command_attributes = 14;
+    }
+}
+
+message StartTimer {
+    string timer_id = 1;
+    google.protobuf.Duration start_to_fire_timeout = 2;
+}
+
+message ScheduleActivity {
```

Comment:
This will probably change after we talk about local vs. non-local activities.

Reply:
We can add the local option once we get to implementing it.